lib/diaspora_federation/schemas.rb

```diff
@@ -0,0 +1,17 @@
+require "pathname"
+require "json"
+
+module DiasporaFederation
+  # A helper class to access the JSON schema.
+  module Schemas
+    # federation_entities schema uri
+    FEDERATION_ENTITIES_URI = "https://diaspora.github.io/diaspora_federation/schemas/federation_entities.json".freeze
+
+    # Parsed federation_entities schema
+    def self.federation_entities
+      @federation_entities ||= JSON.parse(
+        Pathname.new(__dir__).join("schemas", "federation_entities.json").expand_path.read
```

Comment:
It returns the absolute path, but I only copied it from [here](https://github.com/diaspora/diaspora/blob/7934c1e969729686bcc964948bac3a381273ae03/lib/node_info.rb#L157-L159). But you're right, it doesn't make a difference here with `.read`, so I removed it.